app/models/category_localization.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+class CategoryLocalization < ActiveRecord::Base
+  belongs_to :category
+
+  validates :locale, presence: true, length: { maximum: 20 }
+  validates :name, presence: true, length: { maximum: 50 }
+  validates :category_id, uniqueness: { scope: :locale }
+
+  after_commit :invalidate_site_cache
+
+  def invalidate_site_cache
+    Site.clear_cache
+  end
+end
+
+# == Schema Information
+#
+# Table name: category_localizations
+#
+#  id          :bigint           not null, primary key
+#  category_id :bigint           not null
+#  locale      :string(20)       not null
+#  name        :string(50)       not null
+#  description :text
```

Comment:
I realized we're missing any sort of length validation on `Category`, so you could put 65kb worth of stuff in here. Maybe we add some reasonable length validation to both?

Reply:
I think Ted meant length validation on `Category#description`